### ISSUE TITLE
Placement of version attribute?

### ISSUE THREAD

**Issue by mjdecker** [srcML/srcML#1328]
I do not remember if we discussed this, however, I do recall talking about language attribute.

Basically, does the version attribute belong on the root unit or file unit or both.

Currently, srcML seems to put it on both.

Currently, srcDiff only puts it on the root unit.

I think it is valid to have a version attribute on both, however, we should probably be able to specify them separately.  Currently, `--src-version` works as stated above.

For example, someone might keep different versions of a file in a single archive or have an archive that is a merger of multiple revisions.  In which case, a version on the root does not make sense, but a version on a file unit does.  Similarly, a version on the root could be used to denote what version of a system the archive is.

In summary, we should allow for both, however, how they are set may need to be changed.
